tests/fixtures/ecma-features/jsx/embedded-tags.src.js

```diff
@@ -1 +1 @@
-<LeftRight left=<a /> right=<b>monkeys /> gorillas</b> />;
```

Comment:
This is a syntax error in the JSX spec. Please see https://github.com/acornjs/acorn-jsx/issues/106 for more details.